tasks/build.sh

```
@@ -165,6 +170,37 @@ cat tasks/pin_search_path.sql >> release/cipherstash-encrypt-protect.sql
 
 
+# v3-only build (design D9): the self-contained eql_v3 surface — schema, SEM
+# types, scalar domains — globbed from src/v3 ONLY. This is the unit the
+# self-containment gate greps; it is the only artifact that can be "free of
+# eql_v2", because the combined variants glob all of src/. It deliberately does
+# NOT append tasks/pin_search_path.sql (D11): that script is eql_v2-coupled
+# (raises if public.eql_v2_encrypted / eql_v2.ste_vec_entry are absent and only
+# ever pins eql_v2 functions), so appending it would both fail a clean v3
+# install and break the self-containment grep.
+find src/v3 -type f -path "*.sql" ! -path "*_test.sql" | while IFS= read -r sql_file; do
+    echo $sql_file
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)